ksa_compliance/patches.txt

```diff
@@ -11,3 +11,4 @@ ksa_compliance.patches._2024_03_21_update_last_attempt_in_additional_fields
 ksa_compliance.patches._2024_06_05_set_cli_setup_to_manual
 ksa_compliance.patches._2024_06_13_remove_custom_fields_from_sales_invoice #2024-06-13 11:25
 ksa_compliance.patches._2024_07_08_set_siaf_is_latest
+ksa_compliance.patches._2024_08_19_update_old_fatoora_url_in_business_settings #2024-08-19 10:31
```

Comment:
This is one of those cases where the patch should have been pre-sync (since post-sync means the field has been updated to select and left with invalid values until the patch runs). However, we can keep it post-sync after making the changes above (where we'll introduce a new field)